### DISCUSSION TITLE
还不错哟，看起来挺好。

### DISCUSSION THREAD

**Discussion by everwar2017gmail** [halo-dev/halo#4952] [General]
### 描述一下此特性

三点建议：
1、目前用的是ant系列的dumi, 建议参考dumi的优点。
2、建议增加界面风格自定义功能或者提供一些配置参数，可配置。
3、文档的宽度能全屏就好了，22寸或者27寸屏幕打开，中间内容部分特别狭窄。

**Comment by ruibaby** [comment]
感谢支持，但这个看起来似乎不是 issue，我将转移到 Discussion。如果后续有问题反馈和特性建议，建议单独分别提交。
